defaults.go

```diff
@@ -333,6 +333,12 @@ func ReverseAddr(addr string) (arpa string, err error) {
 	if ip == nil {
 		return "", &Error{err: "unrecognized address: " + addr}
 	}
+	return ReverseIP(ip), nil
+}
+
+// ReverseIP returns the in-addr.arpa. or ip6.arpa. hostname of the IP
+// address suitable for reverse DNS (PTR) record lookups.
```

Comment:
This should document that `ip` must be either an IPv4 or IPv6 address.

Reply:
I’d probably also like it to return an empty string if it’s invalid. I think `ip.To6() == nil` should be the right check.

Reply:
When you say so regarding check for `ip.To16`, I feel like the function should return an error as well